clock/clock_test.go

```diff
@@ -24,7 +24,7 @@ func TestAlignedTickLossless(t *testing.T) {
 		if elapsed < expWithin || elapsed > expWithin+lateToleration {
 			t.Fatalf("expected to see tick v0 after %v to %v, but it took %v", expWithin, expWithin+lateToleration, elapsed)
 		}
-	case <-time.After(period + 10*time.Millisecond):
+	case <-time.After(period + 30*time.Millisecond):
```

Comment:
should we just be using `lateToleration` here?

Reply:
yep